library/connect.c

```diff
@@ -64,7 +64,10 @@ int close_conn_internal(struct ziti_conn *conn) {
         FREE(conn->rx);
         conn->flusher->data = NULL;
         uv_close((uv_handle_t *) conn->flusher, free_handle);
-        buffer_cleanup(conn->inbound);
+        if (buffer_available(conn->inbound) > 0) {
+            ZITI_LOG(WARN, "dumping %zd bytes of undelivered data conn[%d]",
```

Comment:
it might be useful to log that the connection is closing in this log msg. line 62 (above) will only log if verbose is on and that seems like relevant information to someone running at warn level.

`closing connection - dumping bytes....`

Reply:
that's implied, no? this message is only logged when connection is closed

Reply:
i would have no clue looking through a log that this is only logged when the connection is closed/closing

Reply:
sure you would. function name(`close_conn_internal()`) is logged as well